### ISSUE TITLE
Merge with bcko/flutter_qrcode_reader?

### ISSUE THREAD

**Issue by tamcy** [PonnamKarthik/QRCodeReaderFlutter#9]
Hi,

Several hours ago I opened an issue in https://github.com/bcko/flutter_qrcode_reader/issues/26 to bring up a problem that there exist two version of qr code reader from the same codebase, which isn't ideal. To avoid confusion I would like to know if you are still actively maintaining this project and whether you have interest to merge with `bcko/flutter_qrcode_reader` so that a single library is maintained. Please refer to the issue for details (also I would like to keep the discussion in that issue, thanks!). This is just priliminary thoughts, and there're still issues to be resolved (currently bcko doesn't have right to publish a package on pub). Also I'm not affiliated with any party, just a user who discovered this issue about two weeks ago. Thanks!

**Comment by ponnamkarthik** [comment]
@tamcy Yes i am still maintaining the project and i am ready to merge my repo :)
